membrane_curvature/lib/mods.py

```diff
@@ -160,27 +54,24 @@ def core_fast_leaflet(z_Ref, leaflet, traj, jump, n_cells, lipid_types,
 
     grid_count = np.zeros([n_cells, n_cells])
 
-    for frame in range(0, traj.n_frames, jump):
```

Comment:
What was the `jump` argument good for?

Reply:
`jump` was to skip frames. I was using `jump = 1` for all the tests so I decided it was not very useful at this point either.